Makefile

```diff
@@ -0,0 +1,80 @@
+# ==============================================================================
+# VARIABLES
+
+# -- Hugo
+THEME = blowfish
+
+# -- Docker
+# Get the current user ID to use for docker run and docker exec commands
+DOCKER_UID = $(shell id -u)
+DOCKER_GID = $(shell id -g)
+DOCKER_USER = $(DOCKER_UID):$(DOCKER_GID)
+COMPOSE = DOCKER_USER=$(DOCKER_USER) docker compose
+COMPOSE_RUN = $(COMPOSE) run --rm
+COMPOSE_RUN_HUGO = $(COMPOSE_RUN) hugo
+
+# ==============================================================================
+# RULES
+
+default: help
+
+# -- Docker
+run: ## run dev server
+	@$(COMPOSE) up --force-recreate -d hugo
+.PHONY: run
+
+stop: ## stop container
+	@$(COMPOSE) stop
+.PHONY: stop
+
+down: ## stop and remove container
+	@$(COMPOSE) down
+.PHONY: down
+
+logs: ## display hugo logs
+	@$(COMPOSE) logs -f hugo
+.PHONY: logs
+
+status: ## an alias for "docker-compose ps"
+	@$(COMPOSE) ps
+.PHONY: status
+
+## -- Git
+git-modules-init: ## Init git modules
+	 git submodule update --init --recursive
+.PHONY: gitmodules-init
```

Comment:
`init` or `update`? Should we create two targets instead?

Reply:
Init as we use `--init` flag.
I replace the command `update-theme` by a command `git-modules-update`

Reply:
Perfect.